### PR TITLE
remove redundant UI code

### PR DESCRIPTION
**Pull request by ManishJu**
Related issue: #XXXX

**Description**

A clear and concise description of what the problem was and how this pull request solves it.

This contribution is funded by [Example](https://example.com).

### CODE REVIEW

File: examples/webgl_morphtargets.html [hunk 1]
```diff
@@ -78,60 +78,20 @@
...
-
-					mesh.morphTargetInfluences[ 2 ] = value;
-
-				} );
-				folder.add( params, 'influence4', 0, 1 ).step( 0.01 ).onChange( function ( value ) {
+				
+				for (let i = 0; i < 8; i++) {
+					const x = (i).toString();
+					params["influence" + x] = 0
+					folder.add(params, "influence" + x, 0, 1).step(0.01).onChange(function (value) {
```
Comment (Mugen87) [inline examples/webgl_morphtargets.html:88]: Although this code is more compact, I feel the previous approach is more expressive and easier to understand (especially for new users).

Comment (ManishJu) [reply examples/webgl_morphtargets.html:88]: The code just has a for loop to  dynamically adds 2 things :

1) The object properties 
2) Its UI element

Nothing fancy

Comment (mrdoob) [reply examples/webgl_morphtargets.html:88]: I agree with @Mugen87 👍

**Comment by mrdoob** [comment]
Sorry, the current code requires less cognitive functioning.

**Pull request closed without merging**